cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/common/Statistic.java

```diff
@@ -4,16 +4,21 @@
 
 package com.linkedin.kafka.cruisecontrol.common;
 
+import com.linkedin.kafka.cruisecontrol.servlet.response.JsonResponseField;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 
 
 public enum Statistic {
+  @JsonResponseField
   AVG("AVG"),
+  @JsonResponseField
   MAX("MAX"),
+  @JsonResponseField
   MIN("MIN"),
-  ST_DEV("STD");
+  @JsonResponseField
+  ST_DEV("ST_DEV");
```

Comment:
Nit: Backwards incompatible change -- is it a breaking change used in json responses?